ui/hooks/flask/useTransactionInsightSnap.js

```diff
@@ -43,16 +44,23 @@ export function useTransactionInsightSnap({
             params: { transaction, chainId, transactionOrigin },
           },
         });
-        setData(d);
+        if (!cancelled) {
+          setData(d);
```

Comment:
While not necessarily an issue with this PR, using single letter variable names like `d` won't work anymore with our latest ESLint config. Can you just name it `data` or something?

Reply:
Done